### DISCUSSION TITLE
etcd snapshots to S3

### DISCUSSION THREAD

**Discussion by Thijmen** [vitobotta/hetzner-k3s#507] [Q&A]
Hi there,

Is there an intermediate solution until https://github.com/vitobotta/hetzner-k3s/pull/471 sees the light to backup the etcd snapshots to S3, without a lot of manual work?

**Comment by vitobotta** [comment]
Hi,

I haven't had a chance to test that PR and others properly yet. For now, if you want, you can build the binary yourself with the changes from that PR until it's merged. It’s not ideal, but it’s a temporary workaround.

The repo includes config for dev container support in VSCode. Once you open the code in a dev container, you’ll have Crystal and all the other tools you need, so building is pretty easy.
